Extensions/Leaderboards/JsExtension.js

```diff
@@ -111,6 +111,37 @@ module.exports = {
       .setIncludeFile('Extensions/Leaderboards/leaderboardstools.js')
       .setFunctionName('gdjs.evtTools.leaderboards.formatPlayerName');
 
+    extension
+      .addAction(
+        'DisplayLeaderboard',
+        _('Display leaderboard'),
+        _('Display leaderboard on top of the game.'),
+        _('Display leaderboard _PARAM1_.'),
+        '',
+        'JsPlatform/Extensions/leaderboard.svg',
+        'JsPlatform/Extensions/leaderboard.svg'
+      )
+      .addCodeOnlyParameter('currentScene', '')
+      .addParameter('leaderboardId', 'Leaderboard', '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Leaderboards/leaderboardstools.js')
+      .setFunctionName('gdjs.evtTools.leaderboards.displayLeaderboard');
+
+    extension
+      .addAction(
+        'CloseLeaderboardView',
+        _('Close leaderboard view'),
```

Comment:
Well, they are a bit different : if you open X leaderboards, you don't open X views: the first leaderboard opens the view, the followings just change the href. So the close leaderboard view seems ok to me.
BUT for sure, it may be not clear for our users. Maybe I can add "If a leaderboard is already open, it will replace it in the view"? in the action `DisplayLeaderboard`?